source/semver.d

```diff
@@ -185,7 +185,7 @@ struct SemVer
         return result;
     }
 
-    private SemVer appendPrerelease0() scope @safe pure nothrow
+    private SemVer appendPrerelease0() @safe pure nothrow
```

Comment:
`return scope` (I think) is the right way to do it here.

Reply:
So either you, or the next DMD is wrong.